src/Main.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
-import HornedBeast from './HornedBeast';
-// import Img from './Img.js';
-
 class Main extends React.Component {
   render() {
     return (
       <>
-        <h2>A Very Special Collection</h2>
-        <HornedBeast title={'Billy the Goat'} img_url={'https://upload.wikimedia.org/wikipedia/commons/b/b1/Goat_face.jpg'} description={'domestic goat'}/>
-        <HornedBeast title={'Title coming soon'} description={'description coming soon'}/>
-        <HornedBeast title={'Title coming soon'} description={'description coming soon'}/>
+        <img src= {this.props.imageURLs[0].image_url} alt={'coming soon'}/>
```

Comment:
You'll want to find a way to make this dynamic so you don't have to hard code in eat bit of JSON data.
HINT: Think about code challenge 02 and if any of those tools will help you go through the data.